packages/button/README.md

```diff
@@ -1,22 +1,18 @@
 #  [`@fremtind/jkl-button`](https://fremtind.github.io/jokul/components/button/)
 
-## Om knapper
-Knapper starter en handling. De oppfordrer brukeren til å gjøre noe. Teksten på knappen forteller hva som vil skje når brukeren velger den. For å gi fokus forstørres knappen når brukeren holder pekeren over eller velger den.
-### Knappetyper
+# Knapper
+Knapper starter en handling. Teksten på knappen forteller hva som vil skje når brukeren trykker på den.
 
--   **Primærknapp** – brukes for hovedhandlingen(e) på en side.
--   **Sekundærknapp** – brukes for handlinger som ikke er like viktige for oss eller brukeren.
--   **Tertiærknapp** - brukes for handlinger som er mindre viktige eller kan føre til uønskede konsekvenser.
+## Knappetyper
 
-En side kan ha en eller flere primærknapper, hvis alle handlingene er like viktige. Der vi gir brukerne flere valg samtidig, skal vi ikke ha mer enn en primærknapp. Da skal den viktigste handlingen ha en primærknapp, mens resten skla være sekundære eller tertiære (hvis det finnes et tredje viktigshetsnivå).
-### Tekst på knapper
-De fleste anekrjente GUI-retningslinjer anbefaler at du holder knappenavn så korte som mulig, helst ett, maks to ord. I et skjema med flere deler for eksempel, kan vi bruke Teksten _Fortsett_ eller eventuelt _Gå videre_ for å få brukeren til å forflytte seg fra en skjemadel til en annen. 
+**Primærknapp:** brukes til den viktigste handlingen på en side -  hovedhandlingen. En side kan noen ganger ha mer enn én hovedhandling, men ikke mange.
 
-#### Eksempler
+**Sekundærknapp:** brukes for handlinger som ikke er like viktige for oss eller brukeren.
 
-Knapper brukes til handlinger som
+**Tertiærknapp:** brukes for handlinger som er mindre viktige, eller som kan gi uønskede konsekvenser.
 
--   Lagre
--   Slett (typisk en tertiærknapp)
--   Fortsett/Gå videre i et skjema
--   Gå til en side
+Disse tre knappetypene har et hierarki. Når brukeren har flere valg samtidig, skal vi kun ha én primærknapp.
+
+## Tekst på knapper
+
+Knappetekst skal være så enkel og kort som mulig og skal indikere en handling. Bruk helst bare to ord. Eksempel: Når vi ber brukeren om å gå til en ny side, kan vi bruke teksten “Fortsett” eller “Gå videre” i stedet for “Gå til neste steg”.
```

Comment:
Tror jeg ville skrevet om til "bruk helst bare ett eller to ord" eller "bruk maks to ord" for å fjerne en evt misforståelse